### DISCUSSION TITLE
There is a delay of about 4 seconds when I want to take the screenshot

### DISCUSSION THREAD

**Discussion by syvshc** [flameshot-org/flameshot#1914] [Q&A] [answered]
Here is my system info
![image](https://user-images.githubusercontent.com/38098591/134199969-674568d2-97e6-44fb-b4c6-74beeb2fa456.png)

And here is a gif when I try to start it, the delay also exists when I choose a line or pen or something else:
![Peek 2021-09-21 23-28](https://user-images.githubusercontent.com/38098591/134200729-7dfbc056-2c7a-4b82-9185-99b115b157b1.gif)

when I use terminal, the problem still exists

![Peek 2021-09-21 23-42](https://user-images.githubusercontent.com/38098591/134208602-5678c48c-75f3-4efd-88cc-ec1a02a28562.gif)

I remember when I install it the first time on this system, it didn't have this delay. I've tried re-install flameshot but it didn't work, so could anyone tell me  how can remove the delay?

I have tested by re-install flameshot, I found that it starts suddenly when I click the icon or `flameshot gui` in terminal ONLY FOR SEVERAL TIMES SCREENSHOTS AFTER INSTALL. And then the delay began longer and longer, until reach the longest, 4 seconds perhaps. I really confused about this problem.

P.S. English is not my native language, but Chinese is. If there is something I didn't explain well, please ask me straight.

**Comment by mmahmoudian** [comment, marked as answer]
This is not a good place to report this: the "issues" section is the best place. Here is only for discussions.

I suggest copy the text of this one and open an issue for it.

Also please tell us how you have installed Flameshot and which version (right-click on the tray icon, and then About)

**Reply by syvshc** [reply]
ok, I move it to "issues" section, thanks for you advise
